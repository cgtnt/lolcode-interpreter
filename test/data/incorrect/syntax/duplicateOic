HAI 

BOTH SAEM 1 AN 2, O RLY?
  YA RLY
      NO WAI
    VISIBLE "OK"
OIC
OIC

KTHXBYE
