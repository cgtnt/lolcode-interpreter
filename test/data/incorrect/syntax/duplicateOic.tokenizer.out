@1:T_BEGIN:HAI-@3:T_COMMAND_TERMINATOR:
-@3:T_EQUAL:BOTH SAEM-@3:T_T_INT:1-@3:T_AND:AN-@3:T_T_INT:2-@3:T_COMMAND_TERMINATOR:,-@3:T_IF:O RLY?-@4:T_COMMAND_TERMINATOR:
-@4:T_THEN:YA RLY-@5:T_COMMAND_TERMINATOR:
-@5:T_ELSE:NO WAI-@6:T_COMMAND_TERMINATOR:
-@6:T_WRITE_STDOUT:VISIBLE-@6:T_T_STRING:"OK"-@7:T_COMMAND_TERMINATOR:
-@7:T_END_IF:OIC-@8:T_COMMAND_TERMINATOR:
-@8:T_END_IF:OIC-@10:T_COMMAND_TERMINATOR:
-@10:T_END:KTHXBYE-@11:T_COMMAND_TERMINATOR:
-@11:T_EOF:
